sql_extensions/migrations/special-queries.sql

```diff
@@ -70,8 +109,8 @@ CREATE INDEX inbox_latest_state_by_market_cap ON inbox_events (
     ((data -> 'instantaneous_stats' ->> 'market_cap')::NUMERIC) DESC
 );
 
-CREATE INDEX inbox_latest_state_by_bump_time ON inbox_events (
-    ((data -> 'state_metadata' ->> 'bump_time')::NUMERIC) DESC
+CREATE INDEX inbox_latest_state_by_nonce ON inbox_events (
+    ((data -> 'state_metadata' ->> 'market_nonce')::NUMERIC) DESC
```

Comment:
We need to use `market_nonce` instead of `bump_time` because `bump_time` can have the exact same time but `market_nonce` never will